apps/meteor/server/modules/listeners/listeners.module.ts

```diff
@@ -271,23 +271,13 @@ export class ListenersModule {
 		service.onEvent('banner.enabled', (bannerId): void => {
 			notifications.notifyLoggedInThisInstance('banner-changed', { bannerId });
 		});
-		service.onEvent('queue.agentcalled', (userId, queuename, callerId): void => {
-			notifications.notifyUserInThisInstance(userId, 'agentcalled', { queuename, callerId });
-		});
-		service.onEvent('queue.agentconnected', (userId, queuename: string, queuedcalls: string, waittimeinqueue: string): void => {
-			notifications.notifyUserInThisInstance(userId, 'agentconnected', { queuename, queuedcalls, waittimeinqueue });
-		});
-		service.onEvent('queue.callerjoined', (userId, queuename, callerid, queuedcalls): void => {
-			notifications.notifyUserInThisInstance(userId, 'callerjoined', { queuename, callerid, queuedcalls });
-		});
-		service.onEvent('queue.queuememberadded', (userId, queuename: string, queuedcalls: string): void => {
-			notifications.notifyUserInThisInstance(userId, 'queuememberadded', { queuename, queuedcalls });
-		});
-		service.onEvent('queue.queuememberremoved', (userId, queuename: string, queuedcalls: string): void => {
-			notifications.notifyUserInThisInstance(userId, 'queuememberremoved', { queuename, queuedcalls });
+
+		service.onEvent('voip.events', (userId, data): void => {
+			notifications.notifyUserInThisInstance(userId, 'voip.events', data);
 		});
-		service.onEvent('queue.callabandoned', (userId, queuename: string, queuedcallafterabandon: string): void => {
-			notifications.notifyUserInThisInstance(userId, 'callabandoned', { queuename, queuedcallafterabandon });
+
+		service.onEvent('call.callerhangup', (userId, data): void => {
+			notifications.notifyUserInThisInstance(userId, 'call.hangup', data);
```

Comment:
Any specific reason for changing the event name from call.callerhangup to call.hangup? Everywhere else we are using the same name of the event to notify the user.